dts/arm/nordic/nrf54l09_enga_cpuapp.dtsi

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nordic/nrf54l09.dtsi>
+#include "nrf54l_05_10_15_cpuapp.dtsi"
```

Comment:
I have a PR that enables `dppic` and `ppib` nodes that should be merged today: https://github.com/zephyrproject-rtos/zephyr/pull/82254

Could you update this file and add the same DTS overrides here?

Reply:
it's including `nrf54l_05_10_15_cpuapp.dtsi` which #82254 updates. Do i need to add it something on top of that?

Reply:
Ah, I see. In that case nothing more is needed.